src/components/GameItem.js

```diff
@@ -2,15 +2,28 @@ import React, { Component } from 'react';
 import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
+import { deflate } from 'zlib';
 
 class GameItem extends Component {
+  constructor() {
+    super()
+    this.state = {
+      className: "game-item"
+    };
+  }
+
   static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
   }
 
-  onItemClicked = () => {
-    // Fill this in for Wave 2!
+  onItemClicked = (props) => {
+    if (this.props.type === 'litter') {
+      this.setState({ className: "game-item spotted-litter" });
+      this.props.scored();
```

Comment:
How would you change this code so that you can't score more than once from litter.